File: app/src/main/java/com/doolsjka/kingfile/KingApp.java
package com.doolsjka.kingfile;
import android.app.Application;
import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.os.Build;
public class KingApp extends Application {
    public static final String PREFS = "kf_prefs";
    @Override public void onCreate() {
        super.onCreate();
        if (Build.VERSION.SDK_INT >= 26) {
            NotificationManager nm = getSystemService(NotificationManager.class);
            nm.createNotificationChannel(new NotificationChannel("kf_media","وسائط",NotificationManager.IMPORTANCE_LOW));
        }
    }
}

File: app/src/main/java/com/doolsjka/kingfile/KingApp.kt
package com.doolsjka.kingfile

import android.app.Application
import android.app.NotificationChannel
import android.app.NotificationManager
import android.os.Build

class KingApp : Application() {
    companion object {
        const val PREFS = "kf_prefs"
        const val CH_MEDIA = "kf_media"
        const val CH_DOWNLOAD = "kf_download"
        const val CH_GENERAL = "kf_general"
        lateinit var instance: KingApp private set
    }
    override fun onCreate() {
        super.onCreate()
        instance = this
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            val nm = getSystemService(NotificationManager::class.java)
            listOf(
                NotificationChannel(CH_MEDIA, "وسائط", NotificationManager.IMPORTANCE_LOW),
                NotificationChannel(CH_DOWNLOAD, "تحميل", NotificationManager.IMPORTANCE_DEFAULT),
                NotificationChannel(CH_GENERAL, "عام", NotificationManager.IMPORTANCE_DEFAULT)
            ).forEach { nm?.createNotificationChannel(it) }
        }
    }
}
